app/models/auction.rb

```diff
@@ -8,19 +8,21 @@ class Auction < ActiveRecord::Base
   enum type: { single_bid: 0, multi_bid: 1 }
   enum awardee_paid_status: { not_paid: 0, paid: 1 }
   enum published: { unpublished: 0, published: 1 }
+  enum purchase_card: { default: 0, other: 1 }
 
   # Disable STI
   self.inheritance_column = :__disabled
 
+  validate :start_price_equal_to_or_less_than_max_if_not_contracting_officer
+  validates :delivery_due_at, presence: true, if: :published?
+  validates :description, presence: true, if: :published?
   validates :ended_at, presence: true
-  validates :started_at, presence: true
+  validates :purchase_card, presence: true
```

Comment:
the form has a drop down with two options -- defaults to "default". Since Michael originally wanted a boolean, I thought this made the most sense (because in a bool situation it would either be "18f pcard" or "not 18f pcard") but we can discuss more if you like!